tests/test_util_file.c

```diff
@@ -2,11 +2,11 @@
 #undef NDEBUG
 #endif
 #include "strerror_override.h"
-#ifdef WIN32
+#ifdef _WIN32_
```

Comment:
This looks like it has an extra underscore

Reply:
Fixed.